packages/create-pantheon-decoupled-kit/src/templates/tailwindless-next/components/grid.module.css

```diff
@@ -0,0 +1,34 @@
+.gradientPlaceholder {
+  border-radius: var(--1) var(--1) 0 0;
+  background-image: linear-gradient(to bottom, var(--light-blue), var(--blue)); 
+  height: inherit;
+}
+
+.card {
+  cursor: pointer;
+  border-radius: var(--2);
+  border: var(--0_5) var(--light-gray) solid;
+  overflow-x: hidden;
+  /* 
+   * generated by https://tailwind-to-css.vercel.app/
```

Comment:
Cool / slightly horrifying that this exists :)

Reply:
It wasn't as useful as I thought it would be, but it's nice for one-offs like this.